packages/terra-core-docs/src/terra-dev-site/doc/alert/AccessibilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,82 @@
+import { Notice } from "@cerner/terra-docs";
+import { Badge } from 'terra-image/package.json?dev-site-package';
+import Whitespace from '../../common/layout-helpers/Whitespace';
+
+import Hyperlink from 'terra-hyperlink';
+
+<Badge />
+
+# Accessibility Guide for Terra Alert
+
+<Notice variant="important" ariaLevel="2">
+
+Using alerts improperly may prevent some of your users from understanding your UI.
+
+</Notice>
+
+<Whitespace />
+
+## Why is this important?
+
+Alert is a form of notifications with varying criticalities that should inform and assist the user with their task.
+
+> "An alert is an element that displays a brief, important message in a way that attracts the user's attention without interrupting the user's task."
+> _&nbsp;&nbsp;&mdash; excerpt from <Hyperlink href="https://www.w3.org/WAI/ARIA/apg/patterns/alert/" variant="external">ARIA Authoring Practices Guide (APG): Alert Pattern</Hyperlink>_[<sup>[1]</sup>](/components/cerner-terra-core-docs/alert/accessibility-guide#linked-references)
+
+## Accessibility Considerations
+
+### Code Considerations
+
+#### Consistency
+
+The use of an alert notification should be consistent in order to provide the user the best experience.
+
+#### Alert Criticality
+
+The user only expects to be interrupted for notifications (alerts) that are of high criticality.
+Less critical notifications should wait until the current task is complete before presenting to the user.
+
+By default, Terra Alert will set the role to `"alert"` for the `alert` type and `"status"` for all other types.
+Terra Alert has an optional role prop but should be used carefully when the default does not meet accessibility or usability expectations.
+
+| Role | Purpose |
+| --- | --- |
+| Alert | See <Hyperlink href="https://w3c.github.io/aria/#alert" variant="external">Alert</Hyperlink>[<sup>[2]</sup>](/components/cerner-terra-core-docs/alert/accessibility-guide#linked-references) - "A type of live region with important, and usually time-sensitive, information." |
+| Status | See <Hyperlink href="https://w3c.github.io/aria/#status" variant="external">Status</Hyperlink>[<sup>[2]</sup>](/components/cerner-terra-core-docs/alert/accessibility-guide#linked-references) - "A type of live region whose content is advisory information for the user but is not important enough to justify an alert, often but not necessarily presented as a status bar." |
```

Comment:
```suggestion
| Status | See <Hyperlink href="https://w3c.github.io/aria/#status" variant="external">WAI-ARIA Status Role</Hyperlink>[<sup>[2]</sup>](/components/cerner-terra-core-docs/alert/accessibility-guide#linked-references) - "A type of live region whose content is advisory information for the user but is not important enough to justify an alert, often but not necessarily presented as a status bar." |
```

Reply:
Added here https://github.com/cerner/terra-core/pull/3789/commits/a7ad60dd792651447e2f6adbbe6d2b994da18b67#diff-2c4657d7282dff34f1995cecbdc9b487c519f018091c12370d911c9c60efd472R42

Reply:
Misread the suggestion, got that updated here https://github.com/cerner/terra-core/pull/3789/commits/6e3ba6b19f3867431af24f034ac3cfba3779bd14#diff-2c4657d7282dff34f1995cecbdc9b487c519f018091c12370d911c9c60efd472R41